### DISCUSSION TITLE
Is there any way to start more than 2 macOS VMs via UTM?

### DISCUSSION THREAD

**Discussion by asdf17128** [utmapp/UTM#4251] [UTM Help (macOS)] [answered]
hello everyone, I bought a Mac studio with 10-Core 128G memory, it is very powerful, but I get an error when I start more than two vms: 

`
No value exists in nvram for the provided key. The maximum supported number of active virtual machines has been reached.
`
I think it's a limit by agreement or something, is there any way to change this limit?

thanks!

**Comment by saagarjha** [comment, marked as answer]
This limit is enforced by macOS.
